spec/services/discussion_service_spec.rb

```diff
@@ -112,6 +110,13 @@
                                  params: discussion_params,
                                  actor: user
       end
+
+      it 'can save reader attributes' do
```

Comment:
Yea.. I really think this feels weird and we should not be saving reader attributes inside the DiscussionService.update method.  Let's make them their own Api calls